src/psyclone/dynamo0p3.py

```diff
@@ -4515,11 +4511,48 @@ def __init__(self, fine_arg, coarse_arg):
         self.cell_map = sym.name
 
         # We have no colourmap information when first created
-        self.colourmap = ""
-        # Name of the variable holding the number of colours
-        self.ncolours_var = ""
-        # Name of the variable holding the last cell of a particular colour
-        self.last_cell_var = ""
+        self._colourmap_symbol = ""
```

Comment:
Should be None rather than a str as will hold a Symbol now? Same for `_ncolours_var_symbol` below?

Reply:
Of course, I totally missed that :(